pkgs/development/python-modules/prometheus-pandas/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ipython
+, matplotlib
+, numpy
+, pandas
+, requests
+}:
+
+buildPythonPackage rec {
+    pname = "prometheus-pandas";
+    version = "0.3.2";
```

```suggestion
    version = "0.3.2";
    pyproject = true;
```
Could you please set `pyproject = true;`?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

---

Done.

---

Having `pyproject = true;` makes the build fail:

```
$ nix build ".#pkgs.python3.pkgs.prometheus-pandas"
error: builder for '/nix/store/ah29ihhmfs2y8h8pa3n8vhv2cjbw695a-python3.11-prometheus-pandas-0.3.2.drv' failed with exit code 1;
       last 10 log lines:
       >   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
       >   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
       >   File "<frozen importlib._bootstrap>", line 1126, in _find_and_load_unlocked
       >   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
       >   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
       >   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
       >   File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
       > ModuleNotFoundError: No module named 'setuptools'
       >
       > ERROR Backend 'setuptools.build_meta:__legacy__' is not available.
       For full logs, run 'nix log /nix/store/ah29ihhmfs2y8h8pa3n8vhv2cjbw695a-python3.11-prometheus-pandas-0.3.2.drv'.
```

---

Could you please add `setuptools` to `nativeBuildInputs`?

---

Done.